Cubical/Data/Int/Base.agda

```diff
@@ -1,36 +1,85 @@
+-- This is the preferred version of the integers in the library. Other
+-- versions can be found in the MoreInts directory.
 {-# OPTIONS --safe #-}
 module Cubical.Data.Int.Base where
 
 open import Cubical.Core.Everything
+open import Cubical.Data.Bool
+open import Cubical.Data.Nat hiding (_+_ ; _·_) renaming (isEven to isEvenℕ ; isOdd to isOddℕ)
 
-open import Cubical.Data.Nat
+infix  8 -_
+infixl 7 _·_
+infixl 6 _+_ _-_
 
-data Int : Type₀ where
-  pos    : (n : ℕ) → Int
-  negsuc : (n : ℕ) → Int
+data ℤ : Type₀ where
+  pos    : (n : ℕ) → ℤ
+  negsuc : (n : ℕ) → ℤ
 
-neg : (n : ℕ) → Int
-neg zero = pos zero
+neg : (n : ℕ) → ℤ
+neg zero    = pos zero
 neg (suc n) = negsuc n
 
-sucInt : Int → Int
-sucInt (pos n)          = pos (suc n)
-sucInt (negsuc zero)    = pos zero
-sucInt (negsuc (suc n)) = negsuc n
+sucℤ : ℤ → ℤ
+sucℤ (pos n)          = pos (suc n)
+sucℤ (negsuc zero)    = pos zero
+sucℤ (negsuc (suc n)) = negsuc n
 
-predInt : Int → Int
-predInt (pos zero)    = negsuc zero
-predInt (pos (suc n)) = pos n
-predInt (negsuc n)    = negsuc (suc n)
+predℤ : ℤ → ℤ
+predℤ (pos zero)    = negsuc zero
+predℤ (pos (suc n)) = pos n
+predℤ (negsuc n)    = negsuc (suc n)
```

Comment:
Since we call addition for integers just `_+_` and so on, would it be more consistent to call these `suc` and `pred`?

Reply:
The clash with `suc` for nat is quite annoying, so I think it's better to have the names separate